src/BaseElement.js

```diff
@@ -189,7 +196,7 @@ class BaseElement extends HTMLElement {
 	 * Will trigger update() when a property was changed
 	 */
 	defineProperties() {
-		Object.keys(this.properties()).forEach((prop) => {
+		Object.keys({ ...this.injectProperties(), ...this.properties() }).forEach((prop) => {
```

Comment:
@quarkus is this the correct order? Or would we expect the injected properties to override the default properties? I'm not sure what is best.

Reply:
I think we were going back an forth with this .. 

I think the order is correct as it prioritizes the (dafault) values of the properties over the default values from the injection map. 

As Properties are more crucial to element-js we decided to not overwrite them ..